.golangci.yml

```diff
@@ -14,7 +14,6 @@ linters-settings:
     packages-with-error-message:
     - k8s.io/apimachinery/pkg/util/clock: 'use k8s.io/utils/clock or k8s.io/utils/clock/testing'
 linters:
-  disable-all: true
```

Comment:
I guess this is ok since we pin the version of the linter we use. 
We should not have new linters be added to our CI without a PR as that would randomly break tests.

Reply:
Exactly my thinking -- when a new `golangci` linter version comes along with new linters (or versions of linters) enabled, that will be caught in the PR that upgrades to the new version(s).